### ISSUE TITLE
Any way to set remote repository in cli mode?

### ISSUE THREAD

**Issue by Sieboldianus** [relekang/python-semantic-release#149]
My situation may appear a little bit untypically, but perhaps this is already implemented:

- we have a closed Gitlab instance that we use for internal software development
- we publish some of our branches/tools to Github, for external code collaboration

So far, I've only released `releases` in Github, because it was not possible in python-semantic-release to setup custom Gitlab instances.

Since `GL_TOKEN` became available, I'd also like to publish releases to the `master` branch on our custom Gitlab instance.

However, python-semantic-release seems to only use the remote that is set as default git remote. Is there any way to have it simultaneously publish releases to both Github and Gitlab using GH_TOKEN (e.g. remote `origin`)  and GL_TOKEN (e.g. remote `gl_origin`)?

**Comment by relekang** [comment]
So is this about having a cli option to the changelog command to be able to push the changelog to gitlab as well?

**Comment by Sieboldianus** [comment]
Yes, either this, or a cli option to select the remote by name (and if it is a gitlab-remote, it will update/release the changelog using the GL_TOKEN; if it is a github-remote,  it will update/release the changelog using the GH_TOKEN).

Does this make sense?

**Comment by relekang** [comment]
That sounds like a good idea.

**Comment by alandtse** [comment]
To add to this, I'd like the ability to identify a second branch to push to. I use a dual branch (dev/master) release system and that sounds like it could just built in to this feature request. There's added complexity allowing for multiple remotes over multiple branches but it seems like a natural extension.

Let me know if you think multiple branch pushes should just be it's own request.

**Comment by codejedi365** [comment]
No update yet, still in the back log

**Comment by codejedi365** [comment]
No change, still in backlog

**Comment by codejedi365** [comment]
My new thought on this is that you can gain what you are looking for with two release configurations. One for GitLab and one for GitHub and then use the [`-c` option](https://python-semantic-release.readthedocs.io/en/latest/commands.html#c-config-file) to select the one you desire for that run. If you have one in the `pyproject.toml` file this will be selected by default in the one job and then your other one could be `github-releaserc.toml`. Unfortunately you will have to keep all the other settings in sync manually since we don't have a cli switch to toggle which repository but this would get you some level of capability.

**Comment by thedannymarsh** [comment]
One way to accomplish this would also be to add a --remote-type command line option, then you could pass "gitlab" or "github", overriding the configuration file's remote.type.

**Comment by codejedi365** [comment]
Still in backlog
